src/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const mongoose = require('mongoose')
+const YAML = require('yamljs')
+const { connector } = require('swagger-routes-express')
```

Comment:
What do you need `yamljs` and `swagger-routes-express` for?

Reply:
I believe this should be require('yaml') not yamljs which converts the yaml file into JSON. This is then used by the connector function to connect the src/api/app.js (one that wasn't really doing anything) to the JSON. I believe this would ensure that the function I am returning matched the API design??